routes/project/location/load_balancer.rb

```diff
@@ -74,12 +74,14 @@ class Clover
       r.patch api? do
         authorize("LoadBalancer:edit", lb.id)
         request_body_params = validate_request_params(%w[algorithm src_port dst_port health_check_endpoint vms])
-        lb.update(
-          algorithm: request_body_params["algorithm"],
-          src_port: Validation.validate_port(:src_port, request_body_params["src_port"]),
-          dst_port: Validation.validate_port(:dst_port, request_body_params["dst_port"]),
-          health_check_endpoint: request_body_params["health_check_endpoint"]
-        )
+        DB.transaction do
+          lb.update(
+            algorithm: request_body_params["algorithm"],
+            health_check_endpoint: request_body_params["health_check_endpoint"]
+          )
+          lb.ports.first.update(src_port: Validation.validate_port(:src_port, request_body_params["src_port"]),
```

Comment:
If we support multiple ports per load balancer, why are we only updating the first?

Is the idea that customer load balancers only support a single port, but k8s load balancers support multiple ports?  If that's the case, maybe it would make more sense to have separate models for k8s load balancers compared to customer load balancers. Maybe the idea is that customer load balancers will eventually support multiple ports, and we'll update this later at that point?